doc/yaml_configuration.md

```diff
@@ -256,19 +256,22 @@ be included in our package. This can be used for a few different purposes, e.g.:
 
 * Ensure that packages you don't want used in your project cannot be used in a
   `package.yaml` file (e.g., for license reasons)
-* Prevent overriding of a global package like `Cabal`. For more information, see
-  Stackage issue
-  [#4425](https://github.com/commercialhaskell/stackage/issues/4425)
 * When using a custom GHC build, avoid incompatible packages (see this
   [comment](https://github.com/commercialhaskell/stack/pull/4655#issuecomment-477954429)).
 
 ~~~yaml
 drop-packages:
-- Cabal
 - buggy-package
 - package-with-unacceptable-license
 ~~~
 
+!!! note
+
+    Note: in older LTSs, it used to be handy to drop Cabal for reasons listed in Stackage
+    issue [#4425](https://github.com/commercialhaskell/stackage/issues/4425). However, since
+    XXX FIXME XXX, Cabal is not included in resolvers, so dropping Cabal no longer has any
```

Comment:
I think the preferred terminology is snapshots, rather than resolvers. I would also say 'included directly' because Cabal is included indirectly by virtue of being a GHC boot package.